code/api/src/main/scala/com/avast/datadog4s/api/MetricFactory.scala

```diff
@@ -1,13 +1,14 @@
 package com.avast.datadog4s.api
 
-import com.avast.datadog4s.api.metric.{ Count, Timer, UniqueSet }
+import com.avast.datadog4s.api.metric.{ Count, Eventer, Timer, UniqueSet }
 
 trait MetricFactory[F[_]] {
   def histogram: HistogramFactory[F]
   def gauge: GaugeFactory[F]
   def timer(aspect: String, sampleRate: Option[Double] = None): Timer[F]
   def count(aspect: String, sampleRate: Option[Double] = None): Count[F]
   def uniqueSet(aspect: String): UniqueSet[F]
+  def eventer: Eventer[F]
```

Comment:
I don't think Event sending has anything to do with metrics or `MetricsFactory`. I think the right approach is to have event sending completely isolated from metric factory and have it's own interface and `.make()` method